.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "astrodb-template-db"]
```

Comment:
Not for this PR, but I wonder if this makes our testing easier? Right now, GitHub Actions (and our local setup) has to clone astrodb-template-db into a specific directory for use; this submodule implementation could make that automatic.

Reply:
Yes, 100%. I made this point in new issue #136 .